### PR TITLE
Replaced glog with klog logger and constants for info levels

### PR DESCRIPTION
**Pull request by Zyqsempai**
This PR is a 
 - [x] Feature Implementation
 - [ ] Bug Fix
 - [ ] Documentation

**What this PR does / why we need it**:

Replaced glog logger with klog.
Added constants for each info log level.
Replaced yardcoded log leveles with constants.

Fixes https://github.com/kubernetes-incubator/service-catalog/issues/2415

Please leave this checklist in the PR comment so that maintainers can ensure a good PR.

Merge Checklist:
 - [ ] New feature 
   - [ ] Tests
   - [ ] Documentation
 - [ ] SVCat CLI flag
 - [ ] Server Flag for config
   - [ ] Chart changes
   - [ ] removing a flag by marking deprecated and hiding to avoid
         breaking the chart release and existing clients who provide a
         flag that will get an error when they try to update

### CODE REVIEW

**Comment by Zyqsempai** [comment]
cc @carolynvs

File: pkg/common/constant.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package common
+
+const (
+	ErrorDetailsInfoLogLevel = iota
+	DefaultInfoLogLevel
+	StatesInfoLogLevel
+	StatesDetailsInfoLogLevel
+	DebugInfoLogLevel
+	TraceInfoLogLevel
```
Comment (MHBauer) [inline pkg/common/constant.go]: I don't think we need to have "Info" in all of them, and I'm not sure we need "LogLevel" either.

I know I don't want to type any of these ever anyway.

File: pkg/common/constant.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+)
```
Comment (MHBauer) [inline pkg/common/constant.go]: link to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md so we know what the reference is supposed to be.

**Comment by Zyqsempai** [comment]
So guys, what should we do? If you'l check my last PR we will se message about that fact that klog is alredy in master branch, it's funny;))
Since we alredy have step one omplemented, what should we do next? Use constants? or first of all move all info levels to range(0-5) from convention, coz in master just glog replaced with klog

**Comment by jberkhahn** [comment]
What? If you rebased this PR on master, it would eliminate the glog-klog parts, leaving it with just moving things to constants.

**Comment by Zyqsempai** [comment]
@jberkhahn Sounds good for me, let's wait what @MHBauer will say.

**Comment by MHBauer** [comment]
close #2559 and rebase this on master so we can delete the branch.

**Comment by MHBauer** [comment]
closing so I can delete the branch.

**Pull request closed without merging**